### PR TITLE
Include information when items not found

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/locales/languages/en.json [hunk 1]
```diff
@@ -195,7 +195,8 @@
     "export_success": "Account exported",
     "account_removed": "Account Removed",
     "unable_to_remove_account": "Unable to remove the account",
-    "buy": "Buy"
+    "buy": "Buy",
+    "no_account_exist": "No Account exists"
```
Comment (mike-parkhill) [inline src/locales/languages/en.json:199]: I think Account should be lowercase since it's not a proper noun

File: src/locales/languages/en.json [hunk 2]
```diff
@@ -371,6 +372,7 @@
     "did_payment_address": "DID Payment Account",
     "invalid_did_file":"Invalid DID file provided.",
     "invalid_did_password":"Incorrect Password",
-    "duplicate_did":"DID already exist in wallet"
+    "duplicate_did":"DID already exist in wallet",
```
Comment (mike-parkhill) [inline src/locales/languages/en.json:375]: more readable maybe? `That DID already exists in the wallet`

File: src/features/didManagement/__snapshots__/CreateNewDIDScreen.test.js.snap [hunk 3]
```diff
@@ -439,6 +439,7 @@ exports[`CreateNewDIDScreen should render correctly with modal 1`] = `
               DID Payment Account
             </Memo(ForwardRef(FormControlLabel))>
             <CustomSelectInput
+              emptyItemMessage="No Account exists"
```
Comment (maycon-mello) [inline src/features/didManagement/__snapshots__/CreateNewDIDScreen.test.js.snap:442]: should we use the locale `no_account_exist` here?

File: src/components/CustomSelectInput.js [hunk 4]
```diff
@@ -49,6 +51,12 @@ export function CustomSelectInput({
           setContentVisible(false);
         }}>
         <Actionsheet.Content>
+          {items.length === 0 ? (
```
Comment (maycon-mello) [inline src/components/CustomSelectInput.js:54]: We could also write
```js
Boolean(items.length) && (
  <Typography variant="h2" mt={1}>
    {emptyItemMessage}
  </Typography>
)

```

**Review by maycon-mello — COMMENTED** [review]
looks good

File: src/locales/languages/en.json [hunk 5]
```diff
@@ -371,6 +372,7 @@
     "did_payment_address": "DID Payment Account",
     "invalid_did_file":"Invalid DID file provided.",
     "invalid_did_password":"Incorrect Password",
-    "duplicate_did":"DID already exist in wallet"
+    "duplicate_did":"That DID already exist in wallet",
```
Comment (mike-parkhill) [inline src/locales/languages/en.json:375]: should be `exists`

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**